src/components/App.tsx

```diff
@@ -106,8 +126,6 @@ const Routes: React.FC<RouteProps> = () => {
     return (
       <div className="App">
         <Switch>
-          <Route path="/SignUp" component={SignUp} />
-          <Route path="/SignIn" component={SignIn} />
```

Comment:
It looks to me like this PR is removing the routing and component for `/SignIn`, but line 92 in this file still has a redirect to that path. Should that have been updated?

Reply:
I have removed the reference now. I also removed EXTERNAL_AUTH feature flag because all auth is now external and not part of UserOffice